filebeat/module/system/syslog/test/debian-12.journal-expected.json

```diff
@@ -36,6 +37,7 @@
         "log.syslog.facility.code": 0,
         "log.syslog.priority": 6,
         "message": "Console: switching to colour frame buffer device 160x50",
+        "process.name": "",
```

Comment:
It seems most of the examples I got don't have a `process.name` available :man_facepalming: . The tests can be improved on  a later PR.